cmd/generate.go

```diff
@@ -14,7 +14,6 @@ import (
 	"github.com/Azure/agentbaker/pkg/aks-engine/api"
```

Comment:
I think that we can remove all unused files by AKS to save the effort.

Reply:
Yes, that's the goal for code cleanup. If you see some unused code, feel free to remove it or let me know and I'll remove it.